client/analytics/report/orders/config.js

```diff
@@ -27,17 +27,27 @@ export const filters = [
 	{ label: __( 'Advanced Filters', 'wc-admin' ), value: 'advanced' },
 ];
 
+const titleContext =
+	'Sentence fragment, logical, refers to filtering by Order Status, Product, Customer, or ' +
+	'Coupon Code by various criteria. Screen shot for context: ' +
+	'https://cloudup.com/cSsUY9VeCVJ';
+
+const ruleContext =
+	'Sentence fragment, logical, "Is/Is Not/Includes/Excludes" refers to searching for ' +
+	'orders matching a chosen Order Status, Product, Customer, or Coupon Code. ' +
+	'Screen shot for context: https://cloudup.com/cSsUY9VeCVJ';
```

Comment:
Do you think these should be applied to each translation individually, so it'll correspond to the correct object type/string combo?

> Sentence fragment, logical, "Is" refers to searching for orders matching a chosen order status. Screenshot for context: https://cloudup.com/cSsUY9VeCVJ

> Sentence fragment, logical, "Is Not" refers to searching for orders that don't match a chosen order status. Screenshot for context: https://cloudup.com/cSsUY9VeCVJ

> Sentence fragment, logical, "Includes" refers to orders including a given product(s). Screenshot for context: https://cloudup.com/cSsUY9VeCVJ

etc